{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ExcessiveEmptyDataFrames","kind":"identifier"}],"platforms":["Linux"],"languages":["swift"]}]}],"topicSections":[{"identifiers":["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/ExcessiveEmptyDataFrames\/==(_:_:)"],"title":"Operators"},{"identifiers":["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/ExcessiveEmptyDataFrames\/init()"],"title":"Initializers"},{"identifiers":["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/ExcessiveEmptyDataFrames\/location"],"title":"Instance Properties"},{"identifiers":["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/ExcessiveEmptyDataFrames\/Equatable-Implementations"],"generated":true,"title":"Default Implementations"}],"kind":"symbol","identifier":{"url":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/ExcessiveEmptyDataFrames","interfaceLanguage":"swift"},"metadata":{"fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ExcessiveEmptyDataFrames","kind":"identifier"}],"navigatorTitle":[{"text":"ExcessiveEmptyDataFrames","kind":"identifier"}],"modules":[{"name":"NIOHTTP2"}],"role":"symbol","title":"NIOHTTP2Errors.ExcessiveEmptyDataFrames","externalID":"s:8NIOHTTP20A6ErrorsO24ExcessiveEmptyDataFramesV","roleHeading":"Structure","symbolKind":"struct"},"sections":[],"relationshipsSections":[{"identifiers":["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Error","doc:\/\/co.pointfree.VaporRouting\/SQ","doc:\/\/co.pointfree.VaporRouting\/s5ErrorP","doc:\/\/co.pointfree.VaporRouting\/s8SendableP"],"type":"conformsTo","kind":"relationships","title":"Conforms To"}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/niohttp2\/niohttp2errors\/excessiveemptydataframes"]}],"abstract":[{"text":"The remote peer has sent an excessive number of empty DATA frames, which looks like a denial of service","type":"text"},{"text":" ","type":"text"},{"text":"attempt, so the connection has been closed.","type":"text"}],"hierarchy":{"paths":[["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors"]]},"schemaVersion":{"minor":3,"patch":0,"major":0},"references":{"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors/ExcessiveEmptyDataFrames/location":{"fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"location","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"}],"abstract":[{"text":"The location where the error was thrown.","type":"text"}],"role":"symbol","kind":"symbol","title":"location","type":"topic","url":"\/documentation\/niohttp2\/niohttp2errors\/excessiveemptydataframes\/location","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/ExcessiveEmptyDataFrames\/location"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Error":{"kind":"symbol","url":"\/documentation\/niohttp2\/niohttp2error","fragments":[{"kind":"keyword","text":"protocol"},{"kind":"text","text":" "},{"kind":"identifier","text":"NIOHTTP2Error"}],"navigatorTitle":[{"kind":"identifier","text":"NIOHTTP2Error"}],"role":"symbol","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Error","abstract":[],"type":"topic","title":"NIOHTTP2Error"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors/ExcessiveEmptyDataFrames/Equatable-Implementations":{"abstract":[],"url":"\/documentation\/niohttp2\/niohttp2errors\/excessiveemptydataframes\/equatable-implementations","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/ExcessiveEmptyDataFrames\/Equatable-Implementations","title":"Equatable Implementations","kind":"article","type":"topic","role":"collectionGroup"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors/ExcessiveEmptyDataFrames":{"type":"topic","title":"NIOHTTP2Errors.ExcessiveEmptyDataFrames","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/ExcessiveEmptyDataFrames","role":"symbol","kind":"symbol","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ExcessiveEmptyDataFrames","kind":"identifier"}],"navigatorTitle":[{"text":"ExcessiveEmptyDataFrames","kind":"identifier"}],"url":"\/documentation\/niohttp2\/niohttp2errors\/excessiveemptydataframes","abstract":[{"text":"The remote peer has sent an excessive number of empty DATA frames, which looks like a denial of service","type":"text"},{"text":" ","type":"text"},{"text":"attempt, so the connection has been closed.","type":"text"}]},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors/ExcessiveEmptyDataFrames/==(_:_:)":{"kind":"symbol","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/ExcessiveEmptyDataFrames\/==(_:_:)","url":"\/documentation\/niohttp2\/niohttp2errors\/excessiveemptydataframes\/==(_:_:)","role":"symbol","fragments":[{"text":"static","kind":"keyword"},{"kind":"text","text":" "},{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"=="},{"kind":"text","text":" "},{"kind":"text","text":"("},{"preciseIdentifier":"s:8NIOHTTP20A6ErrorsO","kind":"typeIdentifier","text":"NIOHTTP2Errors"},{"kind":"text","text":"."},{"preciseIdentifier":"s:8NIOHTTP20A6ErrorsO24ExcessiveEmptyDataFramesV","kind":"typeIdentifier","text":"ExcessiveEmptyDataFrames"},{"kind":"text","text":", "},{"preciseIdentifier":"s:8NIOHTTP20A6ErrorsO","kind":"typeIdentifier","text":"NIOHTTP2Errors"},{"kind":"text","text":"."},{"preciseIdentifier":"s:8NIOHTTP20A6ErrorsO24ExcessiveEmptyDataFramesV","kind":"typeIdentifier","text":"ExcessiveEmptyDataFrames"},{"kind":"text","text":") -> "},{"preciseIdentifier":"s:Sb","kind":"typeIdentifier","text":"Bool"}],"type":"topic","abstract":[],"title":"==(_:_:)"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors/ExcessiveEmptyDataFrames/init()":{"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/ExcessiveEmptyDataFrames\/init()","url":"\/documentation\/niohttp2\/niohttp2errors\/excessiveemptydataframes\/init()","role":"symbol","deprecated":true,"type":"topic","title":"init()","fragments":[{"kind":"identifier","text":"init"},{"kind":"text","text":"()"}],"kind":"symbol","abstract":[]},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors":{"url":"\/documentation\/niohttp2\/niohttp2errors","title":"NIOHTTP2Errors","role":"symbol","type":"topic","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"NIOHTTP2Errors","kind":"identifier"}],"kind":"symbol","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors","abstract":[{"type":"text","text":"Errors that NIO raises when handling HTTP\/2 connections."}],"navigatorTitle":[{"text":"NIOHTTP2Errors","kind":"identifier"}]},"doc://co.pointfree.VaporRouting/SQ":{"identifier":"doc:\/\/co.pointfree.VaporRouting\/SQ","type":"unresolvable","title":"Swift.Equatable"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2":{"url":"\/documentation\/niohttp2","title":"NIOHTTP2","role":"collection","type":"topic","kind":"symbol","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","abstract":[]},"doc://co.pointfree.VaporRouting/s5ErrorP":{"identifier":"doc:\/\/co.pointfree.VaporRouting\/s5ErrorP","type":"unresolvable","title":"Swift.Error"},"doc://co.pointfree.VaporRouting/s8SendableP":{"identifier":"doc:\/\/co.pointfree.VaporRouting\/s8SendableP","type":"unresolvable","title":"Swift.Sendable"}}}